<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupWindow</class>
 <widget class="QWidget" name="StartupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>582</width>
    <height>329</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>582</width>
    <height>329</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Open Project</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Recent Projecs:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="recentProjects">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnBrowse">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Browser</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>New Project</string>
      </attribute>
      <widget class="QLineEdit" name="location">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>411</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1111</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCreate">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="projName">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>411</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">NewProject</string>
       </property>
       <property name="maxLength">
        <number>256</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLocation">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>150</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>57</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Project name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
